c/include/tirex_tracker.h

```diff
@@ -53,65 +53,67 @@ typedef enum tirexMeasure_enum {
 	TIREX_OS_NAME = 0,	 /**< Retrieve the name of the operating system (EnvInfo). */
 	TIREX_OS_KERNEL = 1, /**< Retrieve the version of the kernel (EnvInfo). */
 
+	TIREX_TIME_START = 2, /**< Timestamp when the tracking was started. */
+	TIREX_TIME_STOP = 3,  /**< Timestamp when the tracking was started. */
 	/**
 	 * @brief Measure the "real" (wall clock) time in milliseconds that elapsed between tirexStartTracking and
 	 * tirexStopTracking (Measurement).
 	 */
-	TIREX_TIME_ELAPSED_WALL_CLOCK_MS = 2,
+	TIREX_TIME_ELAPSED_WALL_CLOCK_MS = 4,
 	/**
 	 * @brief Measure the time in milliseconds that the program spend in user mode between tirexStartTracking and
 	 * tirexStopTracking (Measurement).
 	 */
-	TIREX_TIME_ELAPSED_USER_MS = 3,
+	TIREX_TIME_ELAPSED_USER_MS = 5,
 	/**
 	 * @brief Measure the time in milliseconds that the program spend in kernel mode between tirexStartTracking and
 	 * tirexStopTracking (Measurement).
 	 */
-	TIREX_TIME_ELAPSED_SYSTEM_MS = 4,
-
-	TIREX_CPU_USED_PROCESS_PERCENT = 5,
-	TIREX_CPU_USED_SYSTEM_PERCENT = 6,
-	TIREX_CPU_AVAILABLE_SYSTEM_CORES = 7,
-	TIREX_CPU_ENERGY_SYSTEM_JOULES = 8,
-	TIREX_CPU_FEATURES = 9,
-	TIREX_CPU_FREQUENCY_MHZ = 10,
-	TIREX_CPU_FREQUENCY_MIN_MHZ = 11,
-	TIREX_CPU_FREQUENCY_MAX_MHZ = 12,
-	TIREX_CPU_VENDOR_ID = 13,
-	TIREX_CPU_BYTE_ORDER = 14,
-	TIREX_CPU_ARCHITECTURE = 15,
-	TIREX_CPU_MODEL_NAME = 16,
-	TIREX_CPU_CORES_PER_SOCKET = 17,
-	TIREX_CPU_THREADS_PER_CORE = 18,
-	TIREX_CPU_CACHES = 19,
-	TIREX_CPU_VIRTUALIZATION = 20,
-
-	TIREX_RAM_USED_PROCESS_KB = 21,
-	TIREX_RAM_USED_SYSTEM_MB = 22,
-	TIREX_RAM_AVAILABLE_SYSTEM_MB = 23,
-	TIREX_RAM_ENERGY_SYSTEM_JOULES = 24,
+	TIREX_TIME_ELAPSED_SYSTEM_MS = 6,
```

Comment:
Let's keep the existing enums untouched (for backwards compatibility) and rather have new numbers for the new measures

Reply:
We don't lose backwards compatibility (in Terms of API compatibility) if we change the underlying enum values since the values are not part of the public API, only the enum names (as named constants) are. Though I see your point beyond 1.0 releases that, for third-party integrability with other languages, we should not change these constants. Though, since we have not had a 1.0 release yet, I would have favored locality for now